benchmarks-api/src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration>
+
+  <Properties>
+    <Property name="patternLayout">%level{length=1} %d{ISO8601} %c{1.} %m [%t]%n</Property>
+  </Properties>
+
+  <Appenders>
+
+    <Routing name="separateBenchmarkLogs">
+      <Routes pattern="$${ctx:benchmarkDir}_$${ctx:benchmarkName}">
+        <!-- This route is chosen if ThreadContext has values for keys: benchmarkDir, benchmarkName. -->
+        <Route>
+          <RollingFile name="Rolling-${ctx:benchmarkDir}-${ctx:benchmarkName}"
+                       fileName="${ctx:benchmarkDir}/${ctx:benchmarkName}.log"
+                       filePattern="${ctx:benchmarkDir}/${ctx:benchmarkName}-%d{yyyy-MM-dd-HH}.log">
```

Comment:
`${ctx:benchmarkName}` is redudant, since `benchmarkDir` is alredy uniquily defines name of the test. Instead of `${ctx:benchmarkName}` just put `benchamarks`  .